code/style.css

```diff
@@ -1,7 +1,183 @@
+
+html {
+  box-sizing: border-box;
+}
+
+body {
+	font-family: 'Poppins', sans-serif;
+	height: 100%;
+	margin: 0;
+	text-align: center;
+  width: 100%;
+  color: #ffffff;
+  display: flex;
+  flex-direction: column;
+  background: #d0c9c9;
+}
+
+h1 {
+  font-size: 32px;
+}
+
+h3 {
+ padding: 10px;
+}
+/**Header**/
+
+.hero {
+    background-image: url("pottery.jpg");
+    background-position: center top;
+    background-size: cover;
+    padding: 20% 0;
+    margin-bottom: 30px;
+    font-family: 'Trebuchet MS', 'Helvetica';
+}
+
+/** main **/
+
+video {
+  display: flex;
+  width: 100%;
+}
+
+/** Signup form **/
+
+.signup-form {
+  background: #395d75;
+  border-radius: 2px;
+  margin: 15px;
+}
+
+.signup-form input {
+  background-color: #708a9c;
+  margin: 10px;
+  border-radius: 2px;
+  flex-direction: row;
+}
+
+.signup-form input:valid {
+  /*Could be green for example*/
+}
+
+.signup-form input:invalid {
+  border: solid rgba(252, 53, 53, 0.562);
+}
+
+.info {
+  display: flex;
+  flex-direction: column;
+}
+
+.container {
+  display: inline-grid;
+}
+
+a {
+  color: white;
+}
+
+button {
+  padding: 10px;
+  border-radius: 3px;
+  margin: 10px;
+  border: solid white;
+  background: #395d75;
+  color: aliceblue;
+}
+
+.btn-dark {
+  background-color: rgba(82, 82, 180, 0.856);
+}
+
+
+/**accordion**/
+.accordion {
+  background: rgb(245, 244, 242);
+  color:rgb(105, 105, 151);
+  border-radius: 2px;
+  margin: 15px;
+}
+
+/* active, .question:hover {
+  background-color: rgba(82, 100, 151, 0.404); 
+} */
```

Comment:
Code that you are not using can be deleted when you are submitting your code as a pull request (as you have done here). Once you submit it as a pull request it should be the final code. I guess you are commenting this out to have it later on, but it you do a commit with this you can check your commits to find the code. So it won't be lost :)